CMakeLists.txt

```diff
@@ -162,3 +164,8 @@ endif()
 
 lug_set_option(LUGBENCH_API_URI ${DEFAULT_LUGBENCH_API_URI} STRING "URI of the Lugbench API")
 add_definitions(/DLUGBENCH_API_URI="${LUGBENCH_API_URI}")
+
+#add api VERSION
+set(DEFAULT_LUGBENCH_API_VERSION "api/v1")
+lug_set_option(LUGBENCH_API_VERSION ${DEFAULT_LUGBENCH_API_VERSION} STRING "Version of the Lugbench API")
```

Comment:
You don't have to use a "default" variable here, I was using one for the API URI because the default changes between debug and release, it's not the case for this variable.